renderer/src/pages/Dashboard.tsx

```diff
@@ -39,10 +39,9 @@ const Dashboard = (): JSX.Element => {
               </p>
             </div>
             <div className="mb-6">
-              <p className="w-fit text-body-3xs text-grayscale-700 uppercase">Total Earnings (coming soon)</p>
-              <p className="w-fit text-header-m font-bold font-number total-earnings" title="total earnings">
-                {totalEarnings > 0 ? totalEarnings : '--'}
-                {totalEarnings > 0 ? <span className="text-header-3xs">FIL</span> : ''}
+              <p className="w-fit text-body-3xs text-grayscale-700 uppercase">Scheduled rewards</p>
+              <p className="w-fit text-header-m font-bold font-number total-earnings" title="scheduled rewards">
+                {scheduledRewards}&nbsp;<span className="text-header-3xs">FIL</span>
```

Comment:
Let's find a better description for this UI field.

We want to convey the information that the user is eligible to receive this amount of FIL in the next reward payout.

How about "pending rewards"?

I think we should also add a tooltip to explain what this means. Proposed text:

> The amount you accrued for running your Station. These funds will be sent to your Station Wallet address at the end of the payout epoch (approximately once a month).

Reply:
Do we have a tooltip component ready?

Reply:
> > Do we have a tooltip component ready?
> Great question! Isn't there a standard HTML element to "display tooltip on hover"?

LOL :) It's enough to set the "title" attribute.

<img width="320" alt="Screenshot 2023-11-07 at 16 28 10" src="https://github.com/filecoin-station/desktop/assets/1140553/1247ddd4-9231-49da-afb9-99ee77bc7284">

Of course, it would be nicer to show an icon like (?) that will make this tooltip easier to discover, but IMO that's "nice to have, no time to implement" right now.

WDYT?

Reply:
Let's add it as a ticket but no need to implement before launch.

Reply:
> Let's find a better description for this UI field.
> 
> We want to convey the information that the user is eligible to receive this amount of FIL in the next reward payout.
> 
> How about "pending rewards"?
> 
> I think we should also add a tooltip to explain what this means. Proposed text:
> 
> > The amount you accrued for running your Station. These funds will be sent to your Station Wallet address at the end of the payout epoch (approximately once a month).

"This is the reward total you have accrued since your last payout. Scheduled earnings will be sent to your Station Wallet approximately once a month, provided you have earned more than the payout threshold"

Reply:
> Let's add the question mark as a ticket but no need to implement before launch.

Done: https://github.com/filecoin-station/desktop/issues/1087